NOTES.md

```diff
@@ -1,3 +1,105 @@
+# Release 2.1.0-beta (Das Boot)
+
+-- issues up-to-date with the 2.1 milestone as of 2015-07-06 23:00 UTC
+-- contributors " " next as of 3dac994 and docs:master as of 0b661c4
+
+Release highlight:
+
+* Automatic failover
```

Comment:
I think we need 1-2 more points here.
Here's one suggestion:
* More flexible administration for servers and tables